nsqd/http.go

```diff
@@ -120,10 +121,22 @@ func (s *httpServer) pingHandler(w http.ResponseWriter, req *http.Request, ps ht
 }
 
 func (s *httpServer) doInfo(w http.ResponseWriter, req *http.Request, ps httprouter.Params) (interface{}, error) {
+	hostname, err := os.Hostname()
+	if err != nil {
+		return nil, http_api.Err{500, err.Error()}
+	}
 	return struct {
-		Version string `json:"version"`
+		Version          string `json:"version"`
+		BroadcastAddress string `json:"broadcast_address"`
+		Hostname         string `json:"hostname"`
+		HTTPPort         int    `json:"http_port"`
+		TCPPort          int    `json:"tcp_port"`
```

Comment:
That sounds reasonable. Since realizing that `/stats` can be very heavy weight i keep thinking of lighter ways to get some of that data. We do have topic & channel filtering there so that certainly helps.